README

```diff
@@ -38,7 +38,7 @@ components at the following locations:
 
 glib-2.23.5     - http://downloads.mysql.com/source/lgpl/glib-2.23.5.tar.gz
 libzip-0.9.3    - http://downloads.mysql.com/source/lgpl/libzip-0.9.3.tar.gz
-paramiko-1.7.6  - http://downloads.mysql.com/source/lgpl/paramiko-1.7.6.tar.gz
+paramiko-1.16.0 - http://downloads.mysql.com/source/lgpl/paramiko-1.16.0.tar.gz
```

Comment:
No; at some point someone will have to upload it. But it seems wrong for the README to point to the wrong version.

-- 
James Brown, currently mobile

> On Jan 15, 2016, at 08:28, Maks3w <notifications@github.com> wrote:
> 
> In README:
> 
> > @@ -38,7 +38,7 @@ components at the following locations:
> >  
> >  glib-2.23.5     - http://downloads.mysql.com/source/lgpl/glib-2.23.5.tar.gz
> >  libzip-0.9.3    - http://downloads.mysql.com/source/lgpl/libzip-0.9.3.tar.gz
> > -paramiko-1.7.6  - http://downloads.mysql.com/source/lgpl/paramiko-1.7.6.tar.gz
> > +paramiko-1.16.0 - http://downloads.mysql.com/source/lgpl/paramiko-1.16.0.tar.gz
> This resource does not exists
> 
> —
> Reply to this email directly or view it on GitHub.
>